Utilities/bin_patch.cpp

```diff
@@ -598,23 +579,25 @@ static std::basic_string<u32> apply_modification(const patch_engine::patch_info&
 		case patch_type::be64:
 		{
 			*reinterpret_cast<be_t<u64, 1>*>(ptr) = static_cast<u64>(p.value.long_value);
+			applied.push_back((offset + 7) & -4); // Two 32-bit locations
 			break;
 		}
 		case patch_type::bef64:
 		{
 			*reinterpret_cast<be_t<u64, 1>*>(ptr) = std::bit_cast<u64, f64>(p.value.double_value);
+			applied.push_back((offset + 7) & -4); // Two 32-bit locations
 			break;
 		}
 		}
 
-		applied.push_back(resval);
+		// Possibly an executable instruction
+		applied.push_back(offset & -4);
```

Comment:
Only be32 is practically used to signify executable instruction. Okay, also be64 can contain two. But other types are more likely to add garbage to the analyser rather than improve it. Especially floats.

Reply:
I mean, other types add 0 to `applied` on purpose

Reply:
Who said BE32 must not be grbage as well? Analyser should deal with anything you throw at it basically including garbage.

Reply:
I fact, I consider changing the patch to be16 as it only meant to change op.uimm16 of ORI/

Reply:
Don't invert question like this. Basically if it works, don't break it.
If it's be16 it already contains instruction, there is no need to inform analyser.

Reply:
FTR I don't agree but there's no point in addressing in this pr.